src/styles/index.scss

```diff
@@ -1,3 +1,71 @@
-body {
-  margin: 0;
+$cell-size: 100px;
+$week-width: ($cell-size * 7) + 6px;
+$gap-size: 1px;
+
+.body {
+  position: relative;
+}
+
+.calendar {
+  display: flex;
+  gap: $gap-size;
+  padding: 10px;
+  flex-wrap: wrap;
+  width: $week-width;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
according to BEM we can't set outer geometry for block. You can make calendar an element of parent  or set calendar position using flex for body